### ISSUE TITLE
Unable to target class items in CSS

### ISSUE THREAD

**Issue by ca1tlin22** [ca1tlin22/dsn1674-assignment-5#1]
Hi Thomas,

I haven't been able to target my class items in CSS — I've run my HTML and CSS through a validator so there don't seem to be any any open brackets or anything.  When I target divs and imgs, it works but it targets all of them.

Thanks for your time,
Caitlin Gagné
ca1tlin22.github.io/dsn1674-assignment-5  

@thomasjbradley

**Comment by thomasjbradley** [comment]
I see the classes in the CSS, but where are they in the HTML?

**Comment by ca1tlin22** [comment]
Oh I must have taken them out — I was trying to put classes on different items to see if it made a difference.  I've re-added the ones around the left and right grain svg files, which I have been trying to work on and have synced.

**Comment by thomasjbradley** [comment]
Did the changes get synced?

**Comment by ca1tlin22** [comment]
Yep! I've re-added the ones around the left and right grain svg files, which I have been trying to work on and have synced.

**Comment by thomasjbradley** [comment]
This is where you mean, right?
https://github.com/ca1tlin22/dsn1674-assignment-5/blob/gh-pages/index.html#L13-L15
I still can't see any classes.

**Comment by ca1tlin22** [comment]
Yeah there — I have it in my html and I synced it, but it hasn't seemed to change it here

**Comment by thomasjbradley** [comment]
Looking at the last commit, made just over 15 minutes ago, here: https://github.com/ca1tlin22/dsn1674-assignment-5/commit/24151abb859803f3d843cec28a4ba473509a9857
There isn't any changes to the HTML.

**Comment by ca1tlin22** [comment]
Okay, so I've been opening the entire assignment-5 folder and the index that comes up isn't the index that's syncing with Github... the index.html that opens when I pull the entire folder in doesn't work, but if I only pull in the index from the folder it's different... I copied the index.html from the file that I was working on into the index that works and have synced that.

**Comment by thomasjbradley** [comment]
Okay, so there was a folder mis-match. But everything looks like it's working now.

**Comment by ca1tlin22** [comment]
Okay, the new html works fine and I can target things now!  Thanks for your help!
